### DISCUSSION TITLE
Добавить больше возможности для кастомизации палитры панели

### DISCUSSION THREAD

**Discussion by ghost** [moonshine-software/moonshine#525] [Ideas]
Сейчас в  layout-page харкодом забит цвет, понимаю что можно css переопределять цвет но если уж добавили цвет кнопок то основное окно было бы не плохо иметь возможность и его ```background-color: rgb(248 250 252 / var(--tw-bg-opacity));``` менять.

**Comment by lee-to** [comment]
В 2.0 уже сделали в 1 версии добавляться не будет

**Comment by ghost** [comment]
Странно я писал во вторую версию, вообщем я имел ввиду для второй версии
